docs/capabilities/debugging.md

```diff
@@ -15,7 +15,7 @@ Config.configure({
 })
 ```
 
-## Debugging Utilities
+## Debugging in `node` environment (recommended)
```

Comment:
Worth triple checking there are no instances of `#debugging-utilities` in our docs

Reply:
Great point, there was only one direct reference in algokit avm debugger extension repo, patched it in this commit (to be merged on main as once debugger is released) -> https://github.com/algorandfoundation/algokit-avm-vscode-debugger/pull/37/commits/95b7a5ec53e2130a81cd65f5f97fc6050b1624fe